src/App/App.tsx

```diff
@@ -87,214 +123,215 @@ class App extends Component {
         <Container fluid={true}>
           <Row id="main-page-row">
             <Col>
+              {<RouteTracker />}
```

Comment:
@kelvin-muchiri should we check that the GA code is provided here? If it's not then we should not be tracking